aci/convert/convert.go

```diff
@@ -68,7 +69,7 @@ func ToContainerGroup(ctx context.Context, aciContext store.AciContext, p types.
 		return containerinstance.ContainerGroup{}, err
 	}
 
-	var containers []containerinstance.Container
+	var ctnrs []containerinstance.Container
```

Comment:
nit: keep name "containers", more readable

Reply:
This was to avoid the conflict with the package name `containers`. My IDE tells me all the time

Reply:
ah, OK, fine